### ISSUE TITLE
missing dependency for zendframework/zend-feed stand-alone

### ISSUE THREAD

**Issue by RobinvdVleuten** [zendframework/zf2#6419]
When I use _zendframework/zend-feed_ standalone;

```
{
    "zendframework/zend-feed": "~2.3"
}
```

I get the following error;

```
PHP Fatal error:  Class 'Zend\ServiceManager\AbstractPluginManager' not found in...
```

Seems there is missing a dependency for the feed component.

**Comment by barryvdh** [comment]
Same her, first got a `Class 'Zend\Http\Client' not found` error. After requiring the zend-http package, I get `Class 'Zend\ServiceManager\AbstractPluginManager' not found`

**Comment by Ocramius** [comment]
@barryvdh what API is causing this failure?

**Comment by barryvdh** [comment]
Zend feed reader, importing an xml url

**Comment by samsonasik** [comment]
I created PR #7179 for it.

**Comment by weierophinney** [comment]
I've closed #7179 as it is not a reasonable solution.

As noted in that issue:

> The problem people are reporting is a documentation issue: if you use it standalone, you need to create your own implementation of Zend\Feed\Reader\ExtensionPluginInterface that will allow resolving the various feed and entry extensions plugins.
> 
> Right now, the default implementation of that is a Zend\ServiceManager\AbstractPluginManager extension, which is why users are reporting that the dependency is missing. Ideally, we should likely provide a default implementation that does not have dependencies, and only have the current ExtensionManager implementation for those who want to leverage the capabilities provided by our ServiceManager component.

I'm adding this to my TODO list for 2.4. In the meantime, take a look at `Zend\Feed\Reader\ExtensionPluginManager` and how it maps extensions to the classes that provide the functionality; from there, it should be trivial to create a standalone `ExtensionManagerInterface` implementation for your needs.

**Comment by weierophinney** [comment]
Closed with #7254

**Comment by harikt** [comment]
Same as reported earlier https://github.com/zendframework/zf2/issues/5414
